chainntnfs/txnotifier.go

```diff
@@ -1770,6 +1766,20 @@ func (n *TxNotifier) NotifyHeight(height uint32) error {
 			confDetails.Block = nil
 		}
 
+		// If the `confDetails` has already been sent before, we'll
+		// skip it and continue processing the next one.
+		if ntfn.dispatched {
```

Comment:
Should we be removing it from the `ntfnsByConfirmHeight` map in this case? Or do we still need to keep it here until the confirmation horizon has passed (re-org safety)?

Reply:
yea i think it's removed in L1790 below